novm/manager.py

```diff
@@ -227,7 +227,7 @@ def state(output):
                 index=1+len(nics)+len(disks),
                 pci=not(nopci),
                 tag="root",
-                tempdir=self._instances.file(str(os.getpid())),
+                tempdir=self._myInstance.file("writable"),
```

Comment:
It doesn't seem like self._myInstance will be defined when this runs, unless I'm misunderstanding?